### PR TITLE
fix wrong check, dont use webkitAudioContext

### PR DESCRIPTION
**Pull request by fippo**
I am not sure what 'wacx' means... also this doesn't work in Firefox at all, not sure what the intent was/is?

### CODE REVIEW

File: src/content/peerconnection/create-offer/js/main.js [hunk 1]
```diff
@@ -49,7 +48,8 @@ function createOffer() {
     }]
   };
   // These constraints confuse Firefox, even if declared as optional.
-  if (webrtcDetectedBrowser !== 'Firefox') {
+  // See https://bugzilla.mozilla.org/show_bug.cgi?id=1184712
```
Comment (jan-ivar) [inline src/content/peerconnection/create-offer/js/main.js]: I think you want https://bugzilla.mozilla.org/show_bug.cgi?id=1098015

Comment (jan-ivar) [inline src/content/peerconnection/create-offer/js/main.js]: On second thought, this is createOffer, so there shouldn't be a problem. Are we sure this is needed?

File: src/content/peerconnection/create-offer/js/main.js [hunk 2]
```diff
@@ -49,7 +48,8 @@ function createOffer() {
...
+  if (webrtcDetectedBrowser !== 'firefox') {
     offerConstraints.optional.push({
```
Comment (jan-ivar) [inline src/content/peerconnection/create-offer/js/main.js]: At some point we should stop referring to RTOfferOptions as "constraints".

**Comment by KaptenJansson** [comment]
wacx means webkit audio context and this demo was originally created before  "other browsers than chrome" were considered I think. Old versions of Chrome (before blink I believe) and Safari used this (not sure if they have fixed it yet). http://stackoverflow.com/a/29373891

**Comment by KaptenJansson** [comment]
Should we just assume that the browsers we care about (WebRTC enabled) use AudioContext() now? If so then LGTM.

File: src/content/peerconnection/create-offer/js/main.js (hunk 2 above)
Comment (KaptenJansson) [inline src/content/peerconnection/create-offer/js/main.js]: I agree, `options` make more sense right?

**Comment by fippo** [comment]
well, webkitAudioContext is deprecated in Chrome even so... :-)

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**